### PR TITLE
Setup Guide information for MacOS rules

### PR DESCRIPTION
**Pull request by shashank-elastic**
## Issues
https://github.com/elastic/ia-trade-team/issues/207

## Summary
- Add Setup Guide information to MacOS Rules
- Refer approved Template guidelines from [here](https://docs.google.com/document/d/1J4ClwoAkhcP4BDTw94Bwb3nC_6kXsl5pu99Um8Ddt58/edit?pli=1)

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/macos/credential_access_access_to_browser_credentials_procargs.toml [hunk 1]
```diff
@@ -22,11 +22,41 @@ risk_score = 73
...
+- Enter a name for the agent policy in "New agent policy name". If other agent policies already exist, you can click the "Existing hosts" tab and select an existing policy instead.
+For more details on Elastic Agent configuration settings, refer to the [helper guide](https://www.elastic.co/guide/en/fleet/8.10/agent-policy.html).
+- Click "Save and Continue".
+- To complete the integration, select "Add Elastic Agent to your hosts" and continue to the next section to install the Elastic Agent on your hosts.
+For more details on Elastic Defend refer to the [helper guide](https://www.elastic.co/guide/en/security/current/install-endpoint.html).
+
+### Auditbeat Setup
+Auditbeat is a lightweight shipper that you can install on your servers to audit the activities of users and processes on your systems. For example, you can use Auditbeat to collect and centralize audit events from the Linux Audit Framework. You can also use Auditbeat to detect changes to critical files, like binaries and configuration files, and identify potential security policy violations.
```
Comment (Aegrah) [inline rules/macos/credential_access_access_to_browser_credentials_procargs.toml:51]:
```suggestion
Auditbeat is a lightweight shipper that can be installed on your servers to audit the activities of users and processes on your systems. For example, Auditbeat collects and centralizes audit events from the Linux Audit Framework. Auditbeat also detects changes to critical files, like binaries and configuration files, and identifies potential security policy violations.
```

File: rules/macos/credential_access_access_to_browser_credentials_procargs.toml [hunk 2]
```diff
@@ -22,11 +22,41 @@ risk_score = 73
...
+
+#### The following steps should be executed in order to add the Auditbeat on a macOS System:
+- To download and install auditbeat on macOS system, select the platform variant as macOS in eacg step and follow the instructions in this [helper guide](https://www.elastic.co/guide/en/beats/auditbeat/current/auditbeat-installation-configuration.html)
```
Comment (Aegrah) [inline rules/macos/credential_access_access_to_browser_credentials_procargs.toml:54]:
```suggestion
- To download and install Auditbeat on macOS system, select the platform variant as macOS in each step and follow the instructions in this [helper guide](https://www.elastic.co/guide/en/beats/auditbeat/current/auditbeat-installation-configuration.html)
```

**Review by Aegrah — APPROVED** [review]
Nice work. 1 typo was identified, and I would personally rewrite the auditbeat section like so. But the rewriting is personal preference. Maybe @approksiu can share her opinion on that matter!

File: rules/macos/credential_access_access_to_browser_credentials_procargs.toml (hunk 1 above)
Comment (shashank-elastic) [reply rules/macos/credential_access_access_to_browser_credentials_procargs.toml:51]: The restructure was discussed with @Aegrah and was decided to keep this the way it is

File: rules/macos/credential_access_access_to_browser_credentials_procargs.toml (hunk 2 above)
Comment (shashank-elastic) [reply rules/macos/credential_access_access_to_browser_credentials_procargs.toml]: The restructure was discussed with @Aegrah and was decided to keep this the way it is

**Comment by shashank-elastic** [comment]
@DefSecSentinel has reviewed this and suggested we remove the auditbeat index for MacOS altogether. 
cc @approksiu making those changes will keep the review template in docs, if we may need it in future.

**Review by DefSecSentinel — APPROVED** [review]
LGTM

**Pull request merged**